tests/unit/load-webpack.ts

```diff
@@ -0,0 +1,145 @@
+import * as assert from 'intern/chai!assert';
+import * as registerSuite from 'intern!object';
+import global from '../../src/global';
+import load from '../../src/load-webpack';
+
+interface WebpackModules {
+	[id: number]: any;
+}
+
+let webpackModules: WebpackModules;
+function setModules(modules?: { [mid: string]: any }) {
+	let nextId = 0;
+	webpackModules = Object.create(null) as WebpackModules;
+
+	if (modules) {
+		const idMap = Object.create(null);
+		Object.keys(modules).forEach((mid: string) => {
+			const id = nextId++;
+			idMap[mid] = { id, lazy: mid.indexOf('bundle!') > -1 };
```

Comment:
sorry for the delay in replying on this. given its seemingly an implementation detail (of some kind of internal registry). I think it would be a nice to have in the future - that wouldn't block this PR.